src/main/java/com/example/homework10/service/MusicServiceImpl.java

```diff
@@ -37,4 +46,15 @@ public Music updateMusic(int id, Music updateMusic) throws Exception {
     public void deleteMusic(int id) {
         musicMapper.deleteMusic(id);
     }
+
+    @Override
+    public int createMusic(Music music) {
+        try {
+            musicMapper.createMusic(music);
+        } catch (Exception e) {
+            throw new MusicDuplicationException("ミュージックは既にあります");
```

Comment:
例外のなかに原因例外を入れておくほうがよいです。
```suggestion
            throw new MusicDuplicationException("ミュージックは既にあります", e);
```

Reply:
例外情報をログに出すときにどこで何が起きたのかトレースしやすくなります。

Reply:
[6f3c96a](https://github.com/yuu3415/homework10/pull/3/commits/6f3c96adfc35535a19bc15aed262b6f1af37704b)
eをつけることで"ミュージックは既にあります"だけでなく、例外の情報も表示するということですね。
承知いたしました。

Reply:
そのとおりです！
が、コミットハッシュ貼り間違えてますかね？

Reply:
申し訳ございません。
PRの前にリプライしてしまっていましたので流れとしてはPRしてからそのコミットハッシュを貼ると言う手順ですね。
[d27fb35](https://github.com/yuu3415/homework10/pull/3/commits/d27fb35fbc973c617572574c63991bb57e9282b0)

Reply:
コミットハッシュあってますね！